src/conda/env_python3_base.yml

```diff
@@ -23,3 +23,8 @@ dependencies:
 - xesmf=0.3.0
 - gsw=3.3
 - metpy=1.1
+- xgcm
+- xhistogram
+- pip
```

Comment:
Specify the package versions for the additions if possible to avoid slowing down the installation.